.cicd/build.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+set -eo pipefail
+. ./.cicd/helpers/general.sh
+
+fold-execute mkdir -p $BUILD_DIR
+
+if [[ $(uname) == 'Darwin' ]]; then
+
+    cd $BUILD_DIR
+    [[ $TRAVIS == true ]] && fold-execute ccache -s
+    fold-execute cmake -DCMAKE_BUILD_TYPE=Release ..
+    fold-execute make -j$JOBS
+
+else # Linux
+
+    MOUNTED_DIR='/workdir'
+    ARGS=${ARGS:-"--rm -v $(pwd):$MOUNTED_DIR"}
+
+    . $HELPERS_DIR/docker-hash.sh
+
+    PRE_COMMANDS=". $MOUNTED_DIR/.cicd/helpers/logging.sh"
+    COMMANDS="cd $MOUNTED_DIR/build && fold-execute cmake -DCMAKE_TOOLCHAIN_FILE=$MOUNTED_DIR/.cicd/helpers/clang.make -DENABLE_TESTS=ON -DCMAKE_BUILD_TYPE=Release .. && fold-execute make -j$JOBS"
+
+    # Docker Commands
+    if [[ $BUILDKITE == true ]]; then
+        fold-execute $CICD_DIR/generate-base-images.sh
```

Comment:
Let's get rid of the `$CICD_DIR` variable in favor of hard-coded paths because, as we discussed in-person, we do not expect these paths to be changing.